test/spec/utils/list_action_spec.rb

```diff
@@ -0,0 +1,96 @@
+require_relative '../spec_helper'
+
+describe Util::ListAction do
+  let(:client) { double Trello::Client }
+  let(:right_now) { Time.now }
+  let(:_) { anything }
+
+  context 'ListAction.all' do
+    let(:board_id) { 'abc123def' }
+    let(:actions) { Util::ListAction.actions(client, board_id, since: right_now.to_s).sort_by(&:date) }
+    let(:response) { [{date: Time.now, data: {card: {}}}].to_json }
+
+    before(:each) { allow(client).to receive(:get).and_return(response) }
+
+    it 'just wants the actions' do
+      actions
```

Comment:
Prepend `actions` call with `ListAction` - took me a few minutes to grok what this was actually calling.